src/huntsman/pocs/scheduler/field.py

```diff
@@ -0,0 +1,142 @@
+from astroplan import FixedTarget
+from astropy.coordinates import SkyCoord
+
+from panoptes.utils.library import load_module
+from panoptes.pocs.base import PanBase
+from huntsman.pocs.utils import dither
+
+
+class AbstractField(PanBase):
+
+    def __init__(self, name, equinox="J2000", frame="icrs", **kwargs):
+        """ Abstract base class for Field objects.
+        Args:
+            name (str): The name of the field (e.g. target name).
+            equinox (str, optional): The equinox, parsed to astropy.coordinates.SkyCoord.
+                Default: J2000.
+            frame (str, option): The frame to be parsed to astropy.coordinates.SkyCoord.
+                Default: icrs.
+            **kwargs: parsed to PanBase.__init__.
+        """
+        super().__init__(**kwargs)
+
+        self.equinox = equinox
+        self.frame = frame
+        self.name = name
+
+        # Prepare the field name
+        self._field_name = self.name.title().replace(' ', '').replace('-', '')
```

Comment:
Do you want to use underscores here instead?

Reply:
I can see why you don't, so fine with this convention as long as you've thought about it.

Reply:
This is copy paste from `pocs`